esp/services/ws_loggingservice/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ include_directories (
      ./..
 )
 
-ADD_DEFINITIONS( -D_USRDLL -DLOGGINGCOMMON_EXPORTS )
+ADD_DEFINITIONS( -D_USRDLL -DLOGGINGCOMMON_EXPORTS -DWSLOGGING_API_LOCAL )
```

Comment:
maybe for another PR, but this could now link logginglib instead of this and compiling in ws_loggingservice_esp.cpp